test/parallel/test-readline-interface.js

```diff
@@ -396,4 +398,28 @@ function isWarned(emitter) {
     });
   });
 
+  var r;
```

Comment:
use more descriptive variables within a { } block to prevent leakage into other future test cases?